Competition2018/src/main/java/competition/subsystems/elevator/commands/ElevatorMaintainerCommand.java

```diff
@@ -1,30 +1,38 @@
 package competition.subsystems.elevator.commands;
+
 import xbot.common.command.BaseCommand;
 import xbot.common.math.PIDFactory;
 import competition.subsystems.elevator.ElevatorSubsystem;
 import com.google.inject.Inject;
+import com.google.inject.Singleton;
+import xbot.common.math.PIDFactory;
 import xbot.common.math.PIDManager;
 
 public class ElevatorMaintainerCommand extends BaseCommand {
-    
+
     ElevatorSubsystem elevator;
     PIDManager pid;
-    
+
     @Inject
     public ElevatorMaintainerCommand(ElevatorSubsystem elevator, PIDFactory pf) {
         this.elevator = elevator;
         pid = pf.createPIDManager("Elevator", 1, 0, 0);
-        pid.setErrorThreshold(0.1);
+        pid.setErrorThreshold(0.01);
```

Comment:
An error of 0.01 inches?! That's really tight!

Let's go for an error of 1 inch to start.

Reply:
done